modules/tensor_mechanics/include/materials/czm/SalehaniIrani3DCTraction.h

```diff
@@ -0,0 +1,37 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "CZMMaterialBase.h"
+class SalehaniIrani3DCTraction;
+template <>
+InputParameters validParams<SalehaniIrani3DCTraction>();
+/**
+ * Implementation of the non-stateful exponential traction separation law
+ * proposed by Salehani, Mohsen Khajeh and Irani, Nilgoon 2018
+ **/
+class SalehaniIrani3DCTraction : public CZMMaterialBase
+{
+public:
+  SalehaniIrani3DCTraction(const InputParameters & parameters);
+
+protected:
+  /// method returning the traction in the interface coordinate system.
```

Comment:
By convention, we do not include doxygen comments for 'override' functions (because it falls back on using the versions in the base class), so delete the comments on line 26 and 29. This is to avoid potential inconsistencies between the base and derived class descriptions of functions.

Reply:
ok, I'll keep this mind